server/routes/matches.js

```diff
@@ -7,98 +7,99 @@ const { check, validationResult } = require('express-validator/check');
 
 const MatchManager = require('../admin/MatchManagement.js');
 
-router.post("/creatematch", 
-[
-	check('hostID', "No host id provided").not().isEmpty()
-], 
-function(req, res, next) {
-	console.log("hello");
-	const errors = validationResult(req);
-	if(!errors.isEmpty()) {
-		return res.status(400).json({errors: errors.array()});
-	}
+router.post("/creatematch",
+	[
+		check('hostID', "No host id provided").not().isEmpty()
+	],
+	function (req, res, next) {
+		console.log("hello");
+		const errors = validationResult(req);
+		if (!errors.isEmpty()) {
+			return res.status(400).json({ errors: errors.array() });
+		}
 
-	const {hostID} = req.body;
+		const { hostID } = req.body;
 
-	try {
-		let gameID = MatchManager.createMatch(hostID);
+		try {
+			let gameID = MatchManager.createMatch(hostID);
 
-		res.json(gameID);
-	} catch (err) {
-		console.error(err.message);
-		res.status(500).send('Server error');
-	}
-});
+			res.json(gameID);
+		} catch (err) {
+			console.error(err.message);
+			res.status(500).send('Server error');
+		}
+	});
 
 
-router.post("/:matchid/joinmatch", 
-[
-	check('userID', 'User ID is required').not().isEmpty(),
-	check('position', "Position required").not().isEmpty(),
-	check('position', "Invalid position").isIn(["RS", "RF", "BS", "BF"])
-],
-function(req, res, next) {
-	console.log("hello");
-	const errors = validationResult(req);
-	if(!errors.isEmpty()) {
-		return res.status(400).json({errors: errors.array()});
-	}
+router.post("/:matchid/joinmatch",
+	[
+		check('userID', 'User ID is required').not().isEmpty(),
+		check('position', "Position required").not().isEmpty(),
+		check('position', "Invalid position").isIn(["RS", "RF", "BS", "BF"])
+	],
+	function (req, res, next) {
+		console.log("hello");
+		const errors = validationResult(req);
+		if (!errors.isEmpty()) {
+			return res.status(400).json({ errors: errors.array() });
+		}
 
-	const {userID, position} = req.body;
-	const matchID = req.params.matchid;
-	//Put the player in the match at the given position.
-	try {
-		const payload = MatchManager.joinMatch(matchID, userID, position);
-		console.log("here");
-		res.json(payload);
-	} catch (err) {
-		console.error(err.message);
-		res.status(500).send('Server error');
-	}
-});
+		const { userID, position } = req.body;
+		const matchID = req.params.matchid;
+		//Put the player in the match at the given position.
+		try {
+			const payload = MatchManager.joinMatch(matchID, userID, position);
+			console.log("here");
+			res.json(payload);
+		} catch (err) {
+			console.error(err.message);
+			res.status(500).send('Server error');
+		}
+	});

```

Comment:
@Flaagrah @haruka-izm @pavel6767 , I've noticed your formatter settings are different from one another's. You'll want to use Prettier with the same configuration to avoid diffs like this.
https://prettier.io/docs/en/configuration.html